src/STEMMUS_SCOPE_exe.m

```diff
@@ -24,8 +24,12 @@ function STEMMUS_SCOPE_exe(config_file, runMode)
                        'KT', ... % Index of current time step
                        'SiteProperties', ... % Site properties (e.g. lat, lon)
                        'fluxes', ... % Atmospheric fluxes
-                       'TT' ... % Soil temperature over depth
-                      }; %#ok
+                       'TT', ... % Soil temperature over depth
+                       'Sim_Theta_U', % Soil moisture over depth, added by Mostafa

```

Comment:
I believe this one is actually `SoilVariables.Theta_UU`. https://github.com/EcoExtreML/STEMMUS_SCOPE/blob/7be4acf6ea2dbf23bd36c2db75ac86c9a66554af/src/STEMMUS_SCOPE.m#L658-L673

The soil temperature could actually also be obtained through the `SoilVariables`struct :thinking: